config/config.go

```diff
@@ -77,7 +77,7 @@ type Config struct {
 	MemQuotaQuery    int64           `toml:"mem-quota-query" json:"mem-quota-query"`
 	EnableStreaming  bool            `toml:"enable-streaming" json:"enable-streaming"`
 	EnableBatchDML   bool            `toml:"enable-batch-dml" json:"enable-batch-dml"`
-	TxnLocalLatches  TxnLocalLatches `toml:"txn-local-latches" json:"txn-local-latches"`
+	TxnLocalLatches  TxnLocalLatches `toml:"-" json:"-"`
```

Comment:
Will it make `-config-check` failed? Maybe we need to add it to https://github.com/pingcap/tidb/blob/7a67ff482b21b7405b582d799215f5ac333fc0ec/config/config.go#L675 .